stacker_blueprints/empire/daemon.py

```diff
@@ -475,6 +484,9 @@ def get_empire_environment(self):
                 Name="EMPIRE_EC2_SUBNETS_PUBLIC",
                 Value=Join(",", Ref("PublicSubnets"))),
             ecs.Environment(
```

Comment:
This empire environment definition needs to be updated at L417 to use the `cloudformation` scheduler (or better yet, templatize it). Currently its using `cloudformation-migration` which will prevents the daemon container from running with newer versions of Empire